libaugrim/src/lib.rs

```diff
@@ -28,7 +28,6 @@
 //!
 //! The specific actions and events differ by algorithm; refer to the algorithm's documentation.
 
-
```

Comment:
This should be a part of the previous commit

Reply:
Fixed.